### PR TITLE
@romidane: Fix bugs introduced by addition of hot reload server.

### PR DESCRIPTION
**Pull request by romidane**
* The addition of the hot reload module excluded node modules from webpack compiler causing issues with module resolutions.
* The hot reload update also removed the ability to pass a port option in the command 'blackjack preview -p 1234' which has now been restored.
* This update removes the need to have a proxy server for hot reloads by using middleware.
* This update also fixes the issue caused by improper configuration of dev server logging which caused  verbose logging during development.

### CODE REVIEW

File: src/preview-server/bootstrap.js [hunk 1]
```diff
@@ -1,38 +1,38 @@
 import path from 'path';
 import express from 'express';
 import webpack from 'webpack';
-import webpackDevServer from 'webpack-dev-server';
 import webpackMiddleware from 'webpack-dev-middleware';
+import webpackHotMiddleware from 'webpack-hot-middleware';
```
Comment (mackstar) [inline src/preview-server/bootstrap.js]: Is this now doubling up?

**Comment by yash0087** [comment]
I am trying to get it working locally in my machine but doesn't seems to work! It seems to re-bundle when you change and save stuffs but doesn't refresh the content. Might be I am missing something?

File: src/commands/init.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import ComponentCreator from '../project-creation/component-creator';
 import StylesCreator from '../project-creation/styles-creator';
 import TestCreator from '../project-creation/test-creator';
 import PreviewCreator from '../project-creation/preview-creator';
+import EntryCreator from '../project-creation/entry-creator';
```
Comment (joshnesbitt) [inline src/commands/init.js]: Maybe we should call this `EntryPointCreator` to be more explicit?

File: src/preview-server/bootstrap.js [hunk 3]
```diff
@@ -64,64 +70,58 @@ export default function bootstrap(options) {
...
+  app.set('view engine', 'ejs');
+  app.set('views', `${BLACKJACK_HOME}/src/preview-server/views`);
   // Serve static files if requested such as images, css, etc.
   app.use('/', express.static('./'));
 
   app.get('/', (req, res) => {
     res.render('index', { template: '', brand: '', component: '' });
   });
 
+  logger.info(`Starting 🌎  server and listening for changes on http://localhost:${SERVER_PORT}.`);
```
Comment (joshnesbitt) [inline src/preview-server/bootstrap.js]: Emoji ftw 👏

**Comment by romidane** [comment]
@yash0087 have you run ``npm install`` and just incase also run ``npm run postinstall`` in the blackjack toolbelt repository.

File: src/preview-server/bootstrap.js (hunk 3 above)
Comment (romidane) [inline src/preview-server/bootstrap.js]: Haha I saw this in a some project long time ago. I just felt right to use it here 💃

**Comment by romidane** [comment]
@yash0087 you mean the Javascript doesn't reload I see.. The styles reload I'll try and fix it 👍

**Comment by romidane** [comment]
@yash0087 It's fixed now.

**Pull request merged**